product_uom/data/product_uom.xml

```diff
@@ -1,11 +1,88 @@
 <?xml version="1.0" encoding="utf-8"?>
-<odoo>
-    <data noupdate="1">
+<odoo noupdate="1">
 
-        <record id="product_uom_calorie" model="product.uom">
-            <field name="category_id" ref="product_category_energy"/>
-            <field name="name">Calorie</field>
-        </record>
+    <!-- Heat / Energy -->
+    <record id="product_uom_calorie" model="product.uom">
+        <field name="category_id" ref="product_category_energy"/>
+        <field name="name">Calorie(s)</field>
+        <field name="uom_type">reference</field>
+        <field name="factor">1.0</field>
+    </record>
+    <record id="product_uom_kilocalorie" model="product.uom">
+        <field name="category_id" ref="product_category_energy"/>
+        <field name="name">Kilocalorie(s)</field>
```

Comment:
@max3903 - I wasn't sure whether you meant an actual calorie, or a kilocalorie in your original unit. I figured it would be best to be specific by adding the kilo calorie in order to reduce possible ambiguity.